pymc_marketing/clv/models/pareto_nbd.py

```diff
@@ -1052,6 +1056,8 @@ def distribution_new_customer_recency_frequency(
             Not required if `data` Dataframe contains a `T` column.
         random_seed : ~numpy.random.RandomState, optional
             Random state to use for sampling.
+        n_samples : int, optional
+            Number of samples to generate. Defaults to 1.
```

Comment:
Defaults to 1_000 right?

Reply:
1000, yeah sorry missed it. will correct it.

Reply:
done!